index.html

```diff
@@ -0,0 +1,156 @@
+                    <li class="cross-icon"><i id="cross-icon" class="cross fa-solid fa-x"></i></li>
+                    <li class="nav"><a href="./index.html">Home</a></li>
+                    <li class="nav"><a href="./about.html">About</a></li>
+                    <li class="nav"><a href="#event">Event</a></li>
+                    <li class="nav"><a href="#partner">Partner</a></li>
+                    <li class="booktick nav"><a href="#book">Book Ticket(s)</a></li>
+                </ul>
+
+            </div>
+        </nav>
+        <div class="music-section">
+            <p class="heading3">"Hello! HallHulla Music Lover!"</p>
+            <h1 class="heading1">SoundHound HallHulla Music Fistival 2022</h1>
+            <p class="pragraph">The Honky Tonk Festival brings together all country music lovers - the hillbilly music
+                lovers, with the aim to expressing a way to celebrating glorious country heritage, culture and
+                traditions. This fest is meant to rejoice our special moments and emotions in our lives with our loved
+                ones, be it as it may, we're all country boys and girls.</p>
+            <h3 class="date-format">2022.11.11(FRI) ~ 12(SAT)</h3>
+            <p class="pragraph2">@ Ryman Auditorium, Nashville, Tennessee and more</p>
+
+        </div>
+
+    </header>
+
+    <div class="event-cards">
+        <h1 class="E-Program">Main Program</h1>
+        <div class="hr"></div>
+        <div class="card-modal">
+            <div class="card">
+                <img class="icon" src="images/lecturer-icon.svg" alt="">
+                <h2 class="card-title">Old Country and Western</h2>
+                <p class="card-desc">
+                    Listen to some good old, vintage, and classic country musicians
+                    play
+                </p>
+            </div>
+            <div class="card">
+                <img class="icon" src="images/business-presentation-icon.svg" alt="">
+                <h2 class="card-title">Tennessee Whiskey</h2>
+                <p class="card-desc">
+                    You have to determine how much whiskey you can take.
+                </p>
+
+            </div>
+            <div class="card">
+                <img class="icon" src="images/discuss-svgrepo-icon.svg" alt="">
+                <h2 class="card-title">Good Modern Country</h2>
+                <p class="card-desc">
+                    See Modern Country boys sing the souls of country music live.
+                </p>
+
+            </div>
+            <div class="card">
+                <img class="icon" src="images/connect-svgrepo-icon.svg" alt="">
+                <h2 class="card-title">Nice Hunting and Fishing</h2>
+                <p class="card-desc">
+                    Some country talents will be selected among attendees.
+                </p>
+
+            </div>
+            <div class="card">
+                <img class="icon" src="images/lecturer-icon.svg" alt="">
```

Comment:
- [ ] Please add an `alt` text to all your `<img>` tags. This is so that the alternative text will be displayed if the image isn't loaded completely and also so that screen readers can easily detect what image it is.